perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/resnet_run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
can we create a similar [setup_container.sh file](https://github.com/GoogleCloudPlatform/gcsfuse/blob/4eeaead8317a378ec2959b2249b957271c9228a0/perfmetrics/scripts/ml_tests/pytorch_dino_model/setup_container.sh) here to install go, build gcsfuse (with log rotation code), mount gcsfuse, update model files and then run the model ?
Also, you can [copy this script inside container](https://github.com/GoogleCloudPlatform/gcsfuse/blob/4eeaead8317a378ec2959b2249b957271c9228a0/perfmetrics/scripts/ml_tests/pytorch_dino_model/Dockerfile#L28) and then [run that script from inside container](https://github.com/GoogleCloudPlatform/gcsfuse/blob/4eeaead8317a378ec2959b2249b957271c9228a0/perfmetrics/scripts/ml_tests/pytorch_dino_model/Dockerfile#L34).

Also remove [this call ](https://github.com/GoogleCloudPlatform/gcsfuse/blob/f4db1eb48354cc2c8fe5ba7db26273292d18e264/perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh#L21)for this change.

Reply:
Done